packages/shylu/fastilu/src/shylu_fastic.hpp

```diff
@@ -194,8 +194,8 @@ class FastICPrec
             Kokkos::parallel_for(nnzL, copyFunc3);
             double t = timer.seconds();
 
-            std::cout << "Symbolic phase complete." << std::endl;
-            std::cout << "Init time: "<< t << "s" << std::endl;
+            //std::cout << "Symbolic phase complete." << std::endl;
```

Comment:
These were some silly cout in experimental code. Thanks for taking care of it. Instead of commenting them out can you guard them in a DEBUG flag ?